astrophot/models/warp_model.py

```diff
@@ -84,21 +85,16 @@ def initialize(self, target=None, parameters=None, **kwargs):
                     for i in range(1, len(new_prof)):
                         if new_prof[i] - new_prof[i - 1] < target.pixel_length.item():
                             new_prof[i] = new_prof[i - 1] + target.pixel_length.item()
-                    parameters[prof_param].set_profile(new_prof)
+                    parameters[prof_param].prof = new_prof
 
         if not (parameters["PA(R)"].value is None or parameters["q(R)"].value is None):
             return
 
-        if parameters["PA(R)"].value is None:
-            parameters["PA(R)"].set_value(
-                np.zeros(len(parameters["PA(R)"].prof)) + target.north,
-                override_locked=True,
-            )
-
-        if parameters["q(R)"].value is None:
-            parameters["q(R)"].set_value(
-                np.ones(len(parameters["q(R)"].prof)) * 0.9, override_locked=True
-            )
+        with Param_Unlock(parameters["PA(R)"]), Param_SoftLimits(parameters["PA(R)"]):
+            if parameters["PA(R)"].value is None:
+                parameters["PA(R)"].value = np.zeros(len(parameters["PA(R)"].prof)) + target.north
+            if parameters["q(R)"].value is None:
+                parameters["q(R)"].value = np.ones(len(parameters["q(R)"].prof)) * 0.9
```

Comment:
Similar question/suggestion about what is 0.9, and documenting it at least in a variable name.  And a similar recognition that this is not a real change in this PR, but the line was touched, so I take the liberty of commenting.

Reply:
In this case I picked `0.9` as a heuristic. I want to start the axis ratio profile at some neutral position, `1` would be ideal since it would represent no change. However I picked 0.9 so that it doesn't hit the parameter boundary of 1 and so there is some signal in the optimizer as to what happens when the `q(R)` value is adjusted. If I started it at 0.999 it might get stuck and take a while to get the parameter out of that extreme border value. 

I'll put something to this effect in the docs